component/input-component/input.css

```diff
@@ -1,35 +0,0 @@
-@import url('/style.css');
- @import url('/component/slidebaar.css');
- @import url('/component/alert/alert.css');
- body{
-     margin: 0px;
-     padding: 0px;
-     font-family: 'Montserrat', sans-serif;
- }
- 
- .input-container input{
-    border-radius: 5px;
-    height: 2rem;
-    width: 15rem;
-    gap: 2rem;
-    /* display: flex; */
-    flex-wrap: wrap;
-    margin-left: 2rem;
-    margin-top: 2rem;
-    align-items: center;
-    
-}
-.input-container{
-    margin-top: 2rem;
-    justify-content: center;
-}
-.input-container input:hover{
-    border-color: var(--primary-color);
-}
-.input-subtitle{
-    margin-left: 4.4rem;
-}
-.box{
-    background-color: rebeccapurple;
```

Comment:
Use hex value for color

Reply:
hey ansh I have fix that problem you can again check